docs/INSTALL.md

```diff
@@ -86,67 +79,73 @@ Here is the list of DC/OS versions supported on dcos-terraform natively:
 
 **Note**: Master DC/OS version is not meant for production use. It is only for CI/CD testing.
 
-To apply the configuration file, you can use this command below.
+To apply the configuration file and you accept all the default variables you can use this command below, otherwise you can continue along with this guide.
```

Comment:
Use the following command to apply the configuration file and to accept all the default variables; otherwise following the instructions in this guide.

Reply:
Done